proposals/NNNN-max-records-per-node.md

```diff
@@ -0,0 +1,116 @@
+<!-- {% raw %} -->
+
+* Proposal: [0025](0025-filename.md)
```

Comment:
You also need to rename the file (it is currently `proposals/NNN-max-records.per-node.md`) and make the link on this line contain the correct filename.